### ISSUE TITLE
Contact 

### ISSUE THREAD

**Issue by lile-tju** [savasozkan/cloud_detection#1]
sorry that i try to contact you by this way.
eh. i want to ask for your kind help about paper " KINSHIPGAN: SYNTHESIZING OF KINSHIP FACES FROM FAMILY PHOTOS BY REGULARIZING A DEEP FACE NETWORK". 
Could u share this paper used validation datasets ?

**Comment by savasozkan** [comment]
Dear Lile-Tju
For train and validation data, we utilized the data available in 'CodaLab - Competition' by using the same configuration (train and validation sets) determined by the organizers. I think the data is still available on the web.
Thanks for your interest to our paper btw.
BestSavas

| 
| 
|  | 
CodaLab - Competition


 |

 |

 |



    On Wednesday, January 30, 2019, 10:58:36 AM GMT+3, lile-tju <notifications@github.com> wrote:  
 
 
sorry that i try to contact you by this way.
eh. i want to ask for your kind help about paper " KINSHIPGAN: SYNTHESIZING OF KINSHIP FACES FROM FAMILY PHOTOS BY REGULARIZING A DEEP FACE NETWORK".
Could u share this paper used validation datasets ?

**Comment by lile-tju** [comment]
Hi, When I go to download the train and validation data in Verification Challenge, that website of The challenge appears ERROR 404 . So could u share ur validation data and ur pretrained model code about this paper, Thank you for your reply!

**Comment by savasozkan** [comment]
I uploaded the validation set and labels to Google Drive as https://drive.google.com/open?id=19d8UXfRYV2KqumFo-P4ZMm9u87Jp11_N
But I have to delete the file within a day.

**Comment by lile-tju** [comment]
thank you very much
